internal/sample-apiserver/pkg/cmd/server/start.go

```diff
@@ -115,7 +115,11 @@ func (o *WardleServerOptions) Config() (*apiserver.Config, error) {
 		return nil, fmt.Errorf("error creating self-signed certificates: %v", err)
 	}
 
-	o.RecommendedOptions.Etcd.StorageConfig.Paging = utilfeature.DefaultFeatureGate.Enabled(features.APIListChunking)
+	// change: allow etcd options to be nil
+	// TODO: this should be reverted after rebasing sample-apiserver onto https://github.com/kubernetes/kubernetes/pull/101106
```

Comment:
should revert the change after https://github.com/kubernetes/kubernetes/pull/101106 is merged in the upstream sample-apiserver